commit 7f95240e92a367d6a57e7f557df2b8e0ea121936 (refs/changes/43/3743/1)
Author: Yuka Poppe <yuka@netbox.org>
Date:   2018-03-30T20:26:58+02:00 (1 year, 6 months ago)
    
    TRITON-289 booter: changed invocations of `menu.BuildMenuLst` to reflect method definition

diff --git a/lib/booter.js b/lib/booter.js
index 5fa83eb..f278a21 100644
--- a/lib/booter.js
+++ b/lib/booter.js
@@ -73,6 +73,7 @@ function main() {
     case 'menu-lst':
         bootparams.getBootParams({
             adminUuid: config.adminUuid,
+            tftpRoot: config.tftpRoot,
             cacheDir: config.cache.dir,
             mac: getArg('MAC address'),
             napi: napi,
@@ -83,7 +84,7 @@ function main() {
                 return console.error(err.code + ': ' + err.message);
             }
 
-            return menu.buildMenuLst(res, config.tftpRoot, function (lst) {
+            return menu.buildMenuLst(res, function (lst) {
                 return console.log(lst);
             });
         });
@@ -94,6 +95,7 @@ function main() {
     case 'boot-ipxe':
         bootparams.getBootParams({
             adminUuid: config.adminUuid,
+            tftpRoot: config.tftpRoot,
             cacheDir: config.cache.dir,
             mac: getArg('MAC address'),
             napi: napi,
@@ -104,7 +106,7 @@ function main() {
                 return console.error(err.code + ': ' + err.message);
             }
 
-            menu.buildIpxeCfg(res, config.tftpRoot, function (cfg) {
+            menu.buildIpxeCfg(res, function (cfg) {
                 return console.log(cfg);
             });
         });
